src/Entity/Item.php

```diff
@@ -77,6 +98,24 @@ public function setId( $id ) {
 	}
 
 	/**
+	 * @since 5.0
```

Comment:
This is deceptive. While the method might not have been defined in this file, it was already in the class by means of inheritance.